config/routes.rb

```diff
@@ -1,2 +1,10 @@
 Rails.application.routes.draw do
+  root 'user_sessions#new'
+  resources :user_sessions, only: [:new, :create, :destroy]
+  resources :users
+  resources :users do
+    member do
+      get :delete # adding the delete action manual, because rails doesn't give it to you.
```

Comment:
why do we need this delete function ? rails already has the destroy function by default right ?